agent/job_runner.go

```diff
@@ -59,13 +60,20 @@ func (r JobRunner) Create() (runner *JobRunner, err error) {
 	// the Buildkite Agent API
 	runner.logStreamer = LogStreamer{MaxChunkSizeBytes: r.Job.ChunksMaxSizeBytes, Callback: r.onUploadChunk}.New()
 
+	timestamp, err := strconv.ParseBool(os.Getenv("BUILDKITE_TIMESTAMP_LINES"))
```

Comment:
Maybe we should check this in `AgentConfiguration`? Or are you thinking that `BUILDKITE_TIMESTAMP_LINES` could be turned on/off from the BK UI at runtime?